### PR TITLE
[Megas for All] Ability fixes, buff, format info

### PR DESCRIPTION
**Pull request by HematiteH2O**
* Makes the Ability Alchemist cause the target to explode and not the user
* Makes the Ability Gravitas work by changing its name to Gravitation (there was conflict with an alias for G-Max Gravitas)
* Adds Close Combat to Kricketune based on a balance change poll from yesterday
* A handful of people asked about having information on the format and a link to our resources added at the start of a battle, so I did that through the ruleset

### CODE REVIEW

File: config/formats.ts [hunk 1]
```diff
@@ -510,10 +510,10 @@ export const Formats: FormatList = [
...
-			'Dialga', 'Dracovish', 'Dragapult', 'Eternatus', 'Genesect', 'Gengar-Mega', 'Giratina', 'Groudon', 'Ho-Oh', 'Kangaskhan-Mega', 'Kyogre', 'Kyurem-Black',
-			'Kyurem-White', 'Landorus-Base', 'Lucario-Mega', 'Lugia', 'Lunala', 'Marshadow', 'Metagross-Mega', 'Mewtwo', 'Naganadel', 'Necrozma-Dawn-Wings',
-			'Necrozma-Dusk-Mane', 'Necrozma-Ultra', 'Palkia', 'Pheromosa', 'Rayquaza', 'Reshiram', 'Salamence-Mega', 'Shaymin-Sky', 'Solgaleo', 'Tornadus-Therian',
+			'Alakazite', 'Arceus', 'Blastoisinite', 'Blaziken', 'Cinderace', 'Darkrai', 'Darmanitan-Galar', 'Deoxys-Attack', 'Deoxys-Base', 'Deoxys-Speed',
+			'Dialga', 'Dracovish', 'Dragapult', 'Eternatus', 'Genesect', 'Gengarite', 'Giratina', 'Groudon', 'Ho-Oh', 'Kangaskhanite', 'Kyogre', 'Kyurem-Black',
+			'Kyurem-White', 'Landorus-Base', 'Lucarionite', 'Lugia', 'Lunala', 'Marshadow', 'Metagrossite', 'Mewtwo', 'Naganadel', 'Necrozma-Dawn-Wings',
+			'Necrozma-Dusk-Mane', 'Necrozma-Ultra', 'Palkia', 'Pheromosa', 'Rayquaza', 'Reshiram', 'Salamencite', 'Shaymin-Sky', 'Solgaleo', 'Tornadus-Therian',
```
Comment (KrisXV) [inline config/formats.ts:516]: these bans are identical, so this change is pointless

**Review by KrisXV — CHANGES_REQUESTED** [review]

File: config/formats.ts (hunk 1 above)
Comment (HematiteH2O) [reply config/formats.ts:516]: They _should be_ identical, but they're not
People have been using these Megas on Showdown without trouble, while the non-Mega bans are working fine
I don't know why
If there's something else I should do instead, tell me
but I can confirm that changing them this way worked on DH and I also can confirm that we can't leave it as-is

Comment (Zarel) [reply config/formats.ts:516]: Aha. Apparently this is because NatDex is hardcoded by name.

Comment (Zarel) [reply config/formats.ts:516]: Fixed in 22f323dfed8b7a87c574; you can revert this now.

Comment (Zarel) [reply config/formats.ts:516]:
```suggestion
			'Alakazam-Mega', 'Arceus', 'Blastoise-Mega', 'Blaziken', 'Cinderace', 'Darkrai', 'Darmanitan-Galar', 'Deoxys-Attack', 'Deoxys-Base', 'Deoxys-Speed',
			'Dialga', 'Dracovish', 'Dragapult', 'Eternatus', 'Genesect', 'Gengar-Mega', 'Giratina', 'Groudon', 'Ho-Oh', 'Kangaskhan-Mega', 'Kyogre', 'Kyurem-Black',
			'Kyurem-White', 'Landorus-Base', 'Lucario-Mega', 'Lugia', 'Lunala', 'Marshadow', 'Metagross-Mega', 'Mewtwo', 'Naganadel', 'Necrozma-Dawn-Wings',
			'Necrozma-Dusk-Mane', 'Necrozma-Ultra', 'Palkia', 'Pheromosa', 'Rayquaza', 'Reshiram', 'Salamence-Mega', 'Shaymin-Sky', 'Solgaleo', 'Tornadus-Therian',
```

Comment (HematiteH2O) [reply config/formats.ts:516]: Oh!! That's fantastic to know (and really weird o.o') - thank you so much for finding that and for fixing it!!

**Pull request merged**